selfdrive/car/__init__.py

```diff
@@ -236,3 +238,8 @@ def update(self, current_value, current_counter):
     self.previous_value = current_value
 
     return self.rate
+
+
+@dataclass
+class CarData:
```

Comment:
I was thinking we also move the car enum into here in the first PR

Reply:
it would be great if it was still easy to specify a platform you want in our tools. for example test_fw_query_on_routes.py or test_car_model.py

Reply:
how about fuzzy search on the CarInfo?